### ISSUE TITLE
Cant trigger (Need to reboot the device)

### ISSUE THREAD

**Issue by mudassirzulfiqar** [NiasSt90/Geofence4Fhem#1]
Hi, it happens to me sometime geoFences work fine event in the background but after one or two days, It doesn't work, And I need to reboot my device, in order to make them work again?

**Comment by NiasSt90** [comment]
hi,
have you tried to re-register the geofences from the main-menu? Are there any error (toast) message from this call?

Currently the app don't re-register the geofences automatically on all needed system-os events [mentioned here](https://developer.android.com/training/location/geofencing#re-register-geofences-only-when-required). This is on my road map....

**Comment by mudassirzulfiqar** [comment]
No, I have generally discussing geoFence with you, I have seen this repo in stackOverflow, So I thought you are expert in GeoFencing, Im using the Sample GeoFence by Google, Some Time GeoFence works in their sample project, and some time didn't, Even when I re register GeoFence it didnt work. But It happens Some time
